components/rms/slurm/SPECS/slurm.spec

```diff
@@ -487,21 +490,7 @@ install -D -m755 contribs/sjstat %{buildroot}/%{_bindir}/sjstat
 
 # 9/8/14 karl.w.schulz@intel.com - provide starting config file
 %if 0%{?OHPC_BUILD}
-head -n -2 $RPM_BUILD_ROOT/%{_sysconfdir}/slurm.conf.example | grep -v ReturnToService > $RPM_BUILD_ROOT/%{_sysconfdir}/slurm.conf.ohpc
-echo "# OpenHPC default configuration" >> $RPM_BUILD_ROOT/%{_sysconfdir}/slurm.conf.ohpc
-# 10/2/18 brad.geltz@intel.com - Enabling the task/affinity plugin to add the --cpu-bind option to srun for GEOPM
-echo "TaskPlugin=task/affinity" >> $RPM_BUILD_ROOT/%{_sysconfdir}/slurm.conf.ohpc
-echo "PropagateResourceLimitsExcept=MEMLOCK" >> $RPM_BUILD_ROOT/%{_sysconfdir}/slurm.conf.ohpc
-echo "JobCompType=jobcomp/filetxt" >> $RPM_BUILD_ROOT/%{_sysconfdir}/slurm.conf.ohpc
-echo "Epilog=/etc/slurm/slurm.epilog.clean" >> $RPM_BUILD_ROOT/%{_sysconfdir}/slurm.conf.ohpc
-echo "NodeName=c[1-4] Sockets=2 CoresPerSocket=8 ThreadsPerCore=2 State=UNKNOWN" >> $RPM_BUILD_ROOT/%{_sysconfdir}/slurm.conf.ohpc
-echo "PartitionName=normal Nodes=c[1-4] Default=YES MaxTime=24:00:00 State=UP Oversubscribe=EXCLUSIVE" >> $RPM_BUILD_ROOT/%{_sysconfdir}/slurm.conf.ohpc
-# 5/5/2020 karl@oden.utexas.edu - enable configless option
-echo "SlurmctldParameters=enable_configless" >> $RPM_BUILD_ROOT/%{_sysconfdir}/slurm.conf.ohpc
-# 11/9/2021 karl@oden.utexas.edu - setup interactive jobs for salloc
-LaunchParameters=use_interactive_step >> $RPM_BUILD_ROOT/%{_sysconfdir}/slurm.conf.ohpc
-# 6/3/16 nirmalasrjn@gmail.com - Adding ReturnToService Directive to starting config file (note removal of variable during above creation)
-echo "ReturnToService=1" >> $RPM_BUILD_ROOT/%{_sysconfdir}/slurm.conf.ohpc
+cp $RPM_BUILD_ROOT/%{_sysconfdir}/slurm.conf.example $RPM_BUILD_ROOT/%{_sysconfdir}/slurm.conf.ohpc
 # 9/17/14 karl.w.schulz@intel.com - Add option to drop VM cache during epilog
 sed -i '/^# No other SLURM jobs,/i \\n# Drop clean caches (OpenHPC)\necho 3 > /proc/sys/vm/drop_caches\n\n#' $RPM_BUILD_ROOT/%{_sysconfdir}/slurm.epilog.clean
```

Comment:
Here is an example of a silent error!
There is no `slurm.epilog.clean` at https://github.com/SchedMD/slurm/tree/slurm-22-05-10-1/etc, so this `sed` command does nothing!

Reply:
The file has been removed 5 years ago with https://github.com/SchedMD/slurm/commits/master/etc/slurm.epilog.clean

Reply:
It still seems to exist:

```
[root@c1 ~]# rpm -qf /etc/slurm/slurm.epilog.clean
slurm-example-configs-ohpc-22.05.10-2.1.ohpc.2.6.2.x86_64
```

Reply:
https://github.com/openhpc/ohpc/blob/15dab81c0d831e768c50d9e5e1e7d98ea202057b/components/rms/slurm/SPECS/slurm.spec#L485

Reply:
Yep!
I see now that it is copied in SOURCES/ - https://github.com/openhpc/ohpc/blob/3.x/components/rms/slurm/SOURCES/slurm.epilog.clean
All is good!
Except that we could move https://github.com/openhpc/ohpc/blob/15dab81c0d831e768c50d9e5e1e7d98ea202057b/components/rms/slurm/SPECS/slurm.spec#L506 directly in the file.